fastlane/Fastfile

```diff
@@ -93,4 +100,31 @@ def replace_in(previous_text, new_text, path)
   sed_regex = 's|' + previous_text + '|' + new_text + '|'
   backup_extension = '.bck'
   sh("sed", '-i', backup_extension, sed_regex, path)
-end
\ No newline at end of file
+end
+
+def check_api_tester_imports_up_to_date
```

Comment:
This was a slightly hacky approach I thought to make sure we don't forget adding api_tests to the `api_tests_import.dart` file. Any thoughts are welcome!

Reply:
it looks great to me. Small nitpick is that I'd probably have the actual api tests live in their own folder within `lib` if possible, so that we don't have to filter out the `main.dart` and `api_tests_import.dart` files and also so that everything in a folder looks very similar